common/djangoapps/third_party_auth/tests/specs/test_testshib.py

```diff
@@ -157,7 +157,8 @@ class TestShibIntegrationTest(SamlIntegrationTestUtilities, IntegrationTestMixin
         'id': 'id_value',
         'firstName': 'firstName_value',
         'idp_name': 'testshib',
-        'attributes': {u'urn:oid:0.9.2342.19200300.100.1.1': [u'myself']}
+        'attributes': {u'urn:oid:0.9.2342.19200300.100.1.1': [u'myself'], 'name_id': '1'},
+        'session_index': '1',
```

Comment:
Ah, I'd been wondering why this was missing despite clearly being added in the code, but failed to notice that we were using test code that we overrode instead of making an actual request.  Nice catch.